security/providers/idcs-mapper/.helidon-oidc-secret

```diff
@@ -0,0 +1 @@
+108d6fd4-de62-4343-aa24-ad59db6f9846
```

Comment:
I don't know what you are doing but you keep adding these files. You should update the top level .gitignore to ignore them.

Reply:
That's the last time I do it. You need to learn how to do that yourself.